Fix glTF coordinate conversion not converting mesh bounds

greeble-dev opened the pull request:
glTF coordinate conversion is applied to mesh assets, but not their bounds:

<img width="1602" height="939" alt="image" src="https://github.com/user-attachments/assets/8eb82e38-309c-446b-8999-4edc1e98b193" />

After the fix:

<img width="1602" height="939" alt="image" src="https://github.com/user-attachments/assets/966344ac-bca5-4020-95d2-878ed141601d" />

## Testing

Tested using a new `scene_viewer` option that I'll file in a separate PR.

```sh
cargo run --example scene_viewer -- "assets/models/faces/faces.glb" --use-model-forward-direction
```

alice-i-cecile reviewed (changes requested):
Can you please add your test example to the `testbeds`? That looks very helpful!

greeble-dev commented:
Added a scene to `testbed_3d`.

<img width="1602" height="939" alt="478691324-ffb40e11-14e4-427f-adfb-5007e6b2cb0a" src="https://github.com/user-attachments/assets/26842272-38d3-4230-9935-1d18e99041dc" />

I've added `M-Deliberate-Rendering-Change` since `testbed_3d` is part of the CI tests, but I'm not actually sure if that's necessary.

CI is failing, but I think that's unrelated to the PR.

alice-i-cecile commented on examples/testbed/3d.rs:
```diff
@@ -340,3 +351,78 @@ mod gizmos {
+        trigger: On<SceneInstanceReady>,
+        mut commands: Commands,
+        children: Query<&Children>,
+        meshes: Query<(), With<Mesh3d>>,
```
That query won't work for hierarchies with skipped connections :)
